create.html

```diff
@@ -57,12 +63,18 @@ <h2>Add question:</h2>
     </main>
     <!-- Navigation -->
     <nav>
-      <ul>
-        <li><a href="./index.html">Home</a></li>
-        <li><a href="./bookmark.html">Bookmark</a></li>
-        <li><a href="./create.html">Create</a></li>
-        <li><a href="./profile.html">Profile</a></li>
-      </ul>
+      <a class="nav-button" href="./index.html" alt="Home">
```

Comment:
As it is totally clear that this is the navigation you don't need to add a comment. Your code should be self-explaining and it shouldn't be necessary to add comments to explain it to others. :-)